### DISCUSSION TITLE
请问下下有办法实现聊天记录合并转发吗？

### DISCUSSION THREAD

**Discussion by 3dian** [ttttupup/wxhelper#472] [Q&A]
我想用机器人合并转发聊天记录给其他人，请问有什么好的办法吗？

**Comment by ttttupup** [comment]
能在pc上手动操作完成的，基本上就可以实现。不能的话就不好说

**Reply by 3dian** [reply]
PC上是可以合并转发聊天记录的（消息上右键--“多选”--“合并转发”），想请教下大佬通过httpAPI 有办法实现吗？
